Gemfile

```diff
@@ -37,18 +37,48 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13'
+  
   gem 'selenium-webdriver'
 end
 
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'web-console', '>= 3.3.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# ========== My Gems ==============
+gem 'bootstrap', '~> 4.0.0'
+gem 'devise'
+gem 'omniauth-google-oauth2'
+gem 'omniauth-twitter'
+gem 'pry'
+gem 'trix'
+gem 'jquery-rails'
+gem 'sqlite3'
+gem 'rspec-rails', '~>3.0'
+gem 'database_cleaner'
+gem 'capybara', '~> 2.13'
+gem 'factory_girl_rails'
+
+# I'm going to use locally database sqlite
+group :development, :test do
+  gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+end
+
+# And for heroku postgress db
+group :production do
+  gem 'pg'
+  gem 'rails_12factor'
+end
+
+group :test do
```

Comment:
if you do not use this group than it's safe to remove it

Reply:
fixed